Open Judge System/OJS.Common/Models/ExecutionStrategyType.cs

```diff
@@ -34,5 +34,6 @@ public enum ExecutionStrategyType
         MySqlPrepareDatabaseAndRunQueries = 16,
         MySqlRunQueriesAndCheckDatabase = 17,
         MySqlRunSkeletonRunQueriesAndCheckDatabase = 18,
+        RubyExecutionStrategy = 32
```

Comment:
I proposed the arrangement in a previous commit, it's by type (first all C# strategies, then all C++ strategies and so on), it makes more sense in my opinion, because if you're gonna search for a strategy it's going to be by type (though arranging for type alphabetically is clearly better if types can still remain in groups) .